### PR TITLE
Add `Icinga Kubernetes`

### PR DESCRIPTION
**Pull request by lippserd**
**NOTE**: This PR depends on #41. Make sure to merge that one first. Commits from that PR are also included here to ease testing.

This PR adds a chart for deploying
[Icinga Kubernetes](https://github.com/Icinga/icinga-kubernetes) and
[Icinga Kubernetes Web](https://github.com/Icinga/icinga-kubernetes-web).

Note that the chart is subject to change as `Icinga Kubernetes` is not
yet released, especially regarding the configuration. At the moment, a
configuration file is mounted from a `ConfigMap` that does not support
secrets, as `Icinga Kubernetes` lacks support for environment variables.
This will change in the near future.

I also chose to depend on the `Bitnami Common Library Chart` as it
provides nice common template helpers which we just copy around
at the moment.

### CODE REVIEW

File: charts/icinga-stack/Chart.yaml
Comment (mocdaniel) [inline charts/icinga-stack/Chart.yaml:1]: Please revert the changes to the release version. We will cut a release for the whole stack in a separate commit once ready.

File: charts/icinga-stack/charts/icingaweb2/values.yaml
Comment (mocdaniel) [inline charts/icinga-stack/charts/icingaweb2/values.yaml:1]: Please revert these changes and open a dedicated PR, so we can better track different changes for release notes etc.

File: charts/icinga-stack/values.yaml [hunk 1]
```diff
@@ -365,8 +365,8 @@ icingaweb2:
 
   image:
     repository: icinga/icingaweb2
-    tag: 2.11.4
-    pullPolicy: IfNotPresent
+    tag: 2.12.1
```
Comment (mocdaniel) [inline charts/icinga-stack/values.yaml:368]: Same here, please don't bump this version in this PR. If it's needed for Icingaweb-Kubernetes to work, split this PR into 

- one for icinga-kubernetes (the daemon)
- the necessary updates to icingaweb for icingaweb-kubernetes to work

**Review by mocdaniel — CHANGES_REQUESTED** [review]

File: charts/icinga-stack/Chart.yaml
Comment (lippserd) [reply charts/icinga-stack/Chart.yaml:1]: Done.

File: charts/icinga-stack/charts/icingaweb2/values.yaml
Comment (lippserd) [reply charts/icinga-stack/charts/icingaweb2/values.yaml:1]: See #41.

File: charts/icinga-stack/values.yaml (hunk 1 above)
Comment (lippserd) [reply charts/icinga-stack/values.yaml:368]: See #41.

**Review by mocdaniel — APPROVED** [review]
Thx for your contribution, LGTM!

**Review by mocdaniel — APPROVED** [review]
